addevents.rst

```diff
@@ -0,0 +1,15 @@
+Add Events
+==========
+
+AddEvent.CreateLink
+-------------------
+A function to create the "Click to Add Calendar" new feature. This function has a lot of parameters, some are optional. 
+You can have a look at them here: https://www.addevent.com/documentation/direct-url-method
+
+Overloads
+~~~~~~~~~
+1. String link = AddEvent.CreateLink(String client, String startdate Integer duration, String title, String description, String location, String service, String calname);
```

Comment:
@eddieirwins 
- there is a missing comma between startdate and Integer duration
- can make startdate startDate
- can you make all the String to string
- can you make Integer => Int

Reply:
actually ignore the String to string comment